SuperblocksHQ/superblocks-lab
# Change EVM getTransactionCount return value to comply with the JSON RPC specification

### Description of the Change

Change `getTransactionCount` return value in _Browser EVM_ to comply with the _JSON RPC_ specifications and consequently work with the _web3_ provider. _BigNumber_ was returned and a value (quantity) was expected instead. Transforming the complex data into a string enables correct message passing via _JSON RPC_.

Reference: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactioncount

### Verification Process
1. Create a new project starting from _Hello World_ template
2. In `app.js`, change `HelloWorld.prototype.getBlockNumber` from calling `getBlockNumber` to `getTransactionCount`:
```
HelloWorld.prototype.getBlockNumber = function(cb) {
    this.web3.eth.getTransactionCount("0xa48f2e0be8ab5a04a5eb1f86ead1923f03a207fd", function(error, result) {
        console.error(error);
        console.log(result);
        cb(error, result);
    });
};
```
3. Deploy
4. Open _Show Preview_
5. Observe the browser console output


### Github Issues

Resolves #341
